src/test/java/com/kodilla/ecommercee/SpringBootJPAIntegrationTest.java

```diff
@@ -28,4 +33,4 @@ void givenGenericEntityRepository_whenSaveAndRetreiveEntity_thenOK() {
         assertTrue(foundEntity.isPresent());
         assertEquals(genericEntity.getTheValue(), foundEntity.get().getTheValue());
     }
-}
```

Comment:
Don't push such changes from your local repos - they will only clutter the main repository image.
If the program automatically adds this file, just cut it out from the commit.